dockerfiles/ubuntu18/build_custom/openvino_cmake.txt

```diff
@@ -2,6 +2,5 @@
 
 -D CMAKE_BUILD_TYPE=Release
 -D ENABLE_PYTHON=ON
--D NGRAPH_PYTHON_BUILD_ENABLE=ON
--D PYTHON_EXECUTABLE=/usr/bin/python3
--D ENABLE_PROFILING_ITT=ON
```

Comment:
Why did you off ITT profiling?

Reply:
This option should be disabled by default, in my opinion.